communication/marketing-team/social-guidelines.md

```diff
@@ -24,19 +24,42 @@ Kubernetes Organization Members with access to account are:
 ## Themes/Topics
 
 - Contributor/SIG blog posts
-- Events
-- KEPs
+- Contributor events
+- KEPs*
+  - see [#KEPs] section for further guidance
 - Process changes
 - Release info, code freeze, et al
 - Accomplishments (reaching 100,000 GitHub stars for example)
 - Significant historical events as related to the project (Kubernetes birthday, previous events, etc.)
+- Steering Committee Announcements
+
+## KEPs
+([Kubernetes Enhancement Proposals])
+
+### Ground Rules  
+- Social Media team can not proactively look for KEPs to amplify.
+- It's not ok to tweet about a KEP that is going to status: implementable until it's been merged.
+  - a good example is user feedback for features that are alpha
+- KEP metadata is found in the kubernetes/enhancements repo only and not on subsequent PRs.
+
+### Process  
+1. KEP requests must come from the author in the metadata; author will apply the area/contributor-comms label
+2. All tweets and content of tweet(s) must be approved explicitly on the KEP in GitHub by author/reviewer/approver listed in the metadata; the author is required and at least one party from the reviewer and/or approver category in the metadata
+3. If there is no meta data on the KEP, you can not amplify it on any social or communication channels
+4. To guide the discussion effectively, it's recommended to link to a discuss.kubernetes.io topic for the KEP unless asked to link to a GitHub issue by the Author.
```

Comment:
I wonder if we need to explicitly add a point for the author to define the `call to action` or `desired outcome` of Tweet when making the request. Whilst pointing them at the discussion is good that may not be the desired outcome from the authors perspective. My concern is we might be leading the community to think an enhancement will ship in a given Kubernetes release and the truth is that even thought a KEP is in implementable state, if it is not code complete by code freeze then it doesn't ship. I personally have been trying to manage expectations around dualstack in Kubernetes because I've tweeted a link to the KEP.

Reply:
I like the desired outcome a lot